lmdeploy/model.py

```diff
@@ -308,30 +310,32 @@ class Puyu(BaseModel):
     AI Laboratory."""
 
     def __init__(self,
+                 meta_instruction='',
```

Comment:
Can we merge `meta_instruction` to `system`?

Reply:
meta instruction will be changed frequently